tests/test_favicon.py

```diff
@@ -201,3 +201,16 @@ def test_request_kwargs(m):
+
+    icons = favicon.get(
+        'http://mock.com/',
+        html=mock_com_html,
+    )
+    assert icons
+    assert icons[0].url == 'http://mock.com/favicon.ico'
+    assert icons[1].url == 'http://mock.com/wp-content/uploads/2014/03/favicon.ico'
+    print(icons)
```

Comment:
Remove `print statement` and at line break to end of file.

Reply:
Done